models/turbine_models/tests/sd_test.py

```diff
@@ -67,6 +67,38 @@
 
 
 class StableDiffusionTest(unittest.TestCase):
```

Comment:
Only testing that we can compile. Can you also check runtime outputs match torch to some tolerance?